pytest_robotframework/_internal/robot_classes.py

```diff
@@ -221,8 +221,15 @@ def start_suite(self, suite: ModelTestSuite):
         suite.resource.imports.library(
             robot_library.__name__, alias=robot_library.__name__
         )
+        # https://github.com/KotlinIsland/basedmypy/issues/618
+        item = cast(Optional[Item], None)
```

Comment:
You can write `"Item | None"` if you want

Reply:
i was on the fence about this for ages but i decided i prefer doing it this way for 2 reasons:
- black won't format quoted types (eg. `"Item|None"` won't be changed to `"Item | None"`)
- neither mypy or ruff has a rule to detect unnecessarily quoted types, so when we drop support for 3.8 it'll be easier to update `Optional[None]` since ruff will point it out